test/TestLinearPremiumPriceOracle.js

```diff
@@ -0,0 +1,63 @@
+const ENS = artifacts.require('@ensdomains/ens/ENSRegistry');
+const BaseRegistrar = artifacts.require('./BaseRegistrarImplementation');
+const DummyOracle = artifacts.require('./DummyOracle');
+const LinearPremiumPriceOracle = artifacts.require('./LinearPremiumPriceOracle');
+
+const namehash = require('eth-ens-namehash');
+const sha3 = require('web3-utils').sha3;
+const toBN = require('web3-utils').toBN;
+
+contract('LinearPremiumPriceOracle', function (accounts) {
+    let priceOracle;
+
+    before(async () => {
+		ens = await ENS.new();
+		registrar = await BaseRegistrar.new(ens.address, namehash.hash('eth'));
+		await ens.setSubnodeOwner('0x0', sha3('eth'), registrar.address);
+		await registrar.addController(accounts[0]);
+
+        // Dummy oracle with 1 ETH == 2 USD
+        var dummyOracle = await DummyOracle.new(toBN(2000000000000000000));
+        // 4 attousd per second for 3 character names, 2 attousd per second for 4 character names,
```

Comment:
Can you put explanation of what `attousd` is?

Reply:
This is described in the docs to `setPrices` on `StablePriceOracle`.

Reply:
yeah, I noticed as I read through